k8s/helm/charts/centrifugo/templates/httproute.yaml

```diff
@@ -16,7 +16,9 @@ spec:
   rules:
     - matches:
         - path:
-            value: /
+            value: /api/
+        - path:
+            value: /connection/websocket
```

Comment:
@NikolayRn I believe we need to expose also the http stream endpoint?

Reply:
Yes @LiorAgnin, we should also allow another paths, i updated about it in related monday task